.github/workflows/cd.yaml

```diff
@@ -33,24 +33,37 @@ jobs:
       - name: Test
         run: npm run test
 
+      - uses: actions/create-github-app-token@v2
+        id: app-token
+        with:
+          app-id: ${{ vars.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Get GitHub App User ID
+        id: get-user-id
+        run: echo "user-id=$(gh api "/users/${{ steps.app-token.outputs.app-slug }}[bot]" --jq .id)" >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ steps.app-token.outputs.token }}
+
+      - name: Configure git
+        run: |
+          git config --global user.name '${{ steps.app-token.outputs.app-slug }}[bot]'
+          git config --global user.email '${{ steps.get-user-id.outputs.user-id }}+${{ steps.app-token.outputs.app-slug }}[bot]@users.noreply.github.com'
+
       - name: Create Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@v1
         with:
           publish: npx changeset publish
           title: "🦋 Release package updates"
           commit: "Bump package version"
-
+          setupGitUser: false
```

Comment:
This ensures the [changesets action](https://github.com/changesets/action/blob/04d574e831923498156e0b2b93152878063203a3/action.yml#L23-L26) uses the git user we've configured within the "Configure git" step.